migrate/20230920_postgres.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:postgres_server) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :updated_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :project_id, :project, type: :uuid, null: false
+      column :location, :text, collate: '"C"', null: false
+      column :server_name, :text, collate: '"C"', null: false, unique: true
+      column :target_vm_size, :text, collate: '"C"', null: false
+      column :target_storage_size_gib, :bigint, null: false
```

Comment:
I guess I understand why you want to keep target_vm_size and target_storage_size_gib here. This is a preparation for future when we implement scale-up? If so, can we leave a comment here about it. I would ideally ask you to leave it out of this patch but it would require a custom migration script when we introduce this capability to migrate the system. 

On the other hand, I'm not sure how soon we'll work on this. I leave the decision to you.

Reply:
We plan to look into this immediately after we have backup because backups are pre-requisite for failover based scale-up/down.